fortex/health/processors/xray_processor.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2022 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+XRAY_Processor Processor
+"""
+from typing import Dict, Set
+import PIL
+from transformers import pipeline
+from forte.data.data_pack import DataPack
+from forte.common.resources import Resources
+from forte.common.configuration import Config
+from forte.processors.base import PackProcessor
+from ft.onto.base_ontology import Classification
+
+
+__all__ = [
+    "XRAY_Processor",
+]
+
+
+class XRAY_Processor(PackProcessor):
+    r"""
+    Implementation of this XRAY_Processor has been based on the fine-tuned
+    version of google/vit-base-patch16-224-in21k
+    on the chest-xray-pneumonia dataset.
+
+    The finetuned model nickmuchi/vit-finetuned-chest-xray-pneumonia (of huggingface transformers)
+    achieves the following results on the evaluation set:
+
+    Loss: 0.1271
+    Accuracy: 0.9551
+
+    Referred link:
+    https://huggingface.co/nickmuchi/vit-finetuned-chest-xray-pneumonia#vit-finetuned-chest-xray-pneumonia
+    """
+
+    def __init__(self):
+        super().__init__()
+        self.extractor = None
+
+    def set_up(self):
+
```

Comment:
Extra line

Reply:
do you mean blank line or self.extractor = None ?

Reply:
Just the blank line, did I mis-label the line? Apologies, if I did.